File: PhatNHSE63348_SE1262/app/src/main/java/com/fpt/phatnhse63348/practical/activity/MainActivity.java
package com.fpt.phatnhse63348.practical.activity;

import android.content.Intent;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;

import com.fpt.phatnhse63348.practical.R;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        Button addStorm = findViewById(R.id.addStorm);
        addStorm.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                startActivity(new Intent(MainActivity.this, AddStormActivity.class));
            }
        });
        Button showListStorm = findViewById(R.id.showListStorm);
        showListStorm.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                startActivity(new Intent(MainActivity.this, ListStormActivity.class));
            }
        });
    }
}

File: kotlin/Day10/app/src/main/java/fpt/phatnh/day10/MainActivity.kt
package fpt.phatnh.day10

import android.content.Context
import android.content.Intent
import android.content.SharedPreferences
import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import android.widget.Toast
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        val sharedPreferences = getSharedPreferences("fpt.phatnh.day10_preferences", Context.MODE_PRIVATE)
        loadPreferenceButton.setOnClickListener {
            startActivity(Intent(this, PreferenceActivity::class.java))
        }
        viewPreferenceButton.setOnClickListener {
            Toast.makeText(this, sharedPreferences.getString("username", "This is the default value"), Toast.LENGTH_LONG).show()
        }
        updatePreferenceButton.setOnClickListener {
            val editor = sharedPreferences.edit()
            editor.putString("username", editText.text.toString())
            editor.commit()
        }
    }
}
